exonum_client/protobuf_loader.py

```diff
@@ -167,6 +192,8 @@ def deinitialize(self) -> None:
             if module.startswith("exonum_modules"):
                 del sys.modules[module]
 
+        getLogger(__name__).debug("Successfully deinitialized ProtobufLoader.")
```

Comment:
Please show reference count at the debug level each time it gets incremented/decremented (so we'll be able to track usage).